### PR TITLE
chore: update cleanup history gql

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by darkskygit** [comment]
* **#7103** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7103?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @darkskygit and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/graphql/src/schema.ts [hunk 1]
```diff
@@ -30,12 +30,39 @@ export interface Scalars {
   DateTime: { input: string; output: string };
   /** The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
   JSON: { input: Record<string, string>; output: Record<string, string> };
+  /** The `JSONObject` scalar type represents JSON objects as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf). */
+  JSONObject: { input: any; output: any };
```
Comment (pengx17) [inline packages/frontend/graphql/src/schema.ts:34]: this seems to be irrelevant. Is it related to the json mode output?

Comment (darkskygit) [reply packages/frontend/graphql/src/schema.ts:34]: used in updateRuntimeConfigs, import automatic by gql-gen

**Review by pengx17 — APPROVED** [review]
